src/components/Title/index.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import styles from "./index.module.css";
+
+const THEMES_VALUES = [
+  styles.redTheme,
+  styles.neonCarrotTheme,
+  styles.greyishBlueTheme,
+];
+
+const TYPE_VALUES = [styles.h1, styles.h2];
+
+export const THEMES = {
+  RED: THEMES_VALUES[0],
+  NEON_CARROT: THEMES_VALUES[1],
+  GREISH_BLUE: THEMES_VALUES[2],
+};
+
+export const TYPE = {
+  H1: TYPE_VALUES[0],
+  H2: TYPE_VALUES[1],
+};
+
+const Title = ({ children, theme, size }) => {
+  const className = `${styles.root} ${theme} ${size}`;
+
+  return <div className={className}>{children}</div>;
```

Comment:
I would use semantic `html`, `h1` and `h2`. Maybe breaking the two into two different components. But if I'm the only one feel free to disregard this comment

Reply:
Can't we put everything into the `Text` component? Use a `variant` prop instead of size that also sets the element

Reply:
@joaojusto  do you mean have a title (component) and a subtite(h2) component?

Reply:
@jfranciscosousa, We could do that, but I don't know if we should do it. Maybe keep them separated makes them simple to use. 🤷‍♂️

Reply:
Personally, I use a `Text` component with a `variant` prop. That prop can be a `h1, h2, h3, etc`. Also a `p` and `small`. I feel like that's pretty straightforward. More code = Larger app :joy:

Reply:
I prefer to use the semantic HTML tag as it's pretty easy to remember, compared to a custom typography scale. It's also important to render the actual semantic HTML on the markup so browsers and SEO bots can actually infer that "hey that's is text and not a div"

Reply:
Might mean smaller code, but it will actually increase complexity, I think it's easier to read the markup if you have something like 
```
<Title>Minho Covid</Title>
<Subtitle>Ajudamos a ajudar</Subtitle>
<Text>Budaerros</Text>
```
than:
```
<Text variant="h1">Minho Covid</Text>
<Text variant="h2">Ajudamos a ajudar</Text>
<Text variant="p">Budaerros</Text>
```
specially when you'll have a bunch of other props like themes. 

But as I said, I don't have strong feelings about it, the important thing as @jfranciscosousa said is rendering semantic markup, as it helps a lot with SEO and accessibility

Reply:
And by breaking these into different components, you don't need a `size` prop

Reply:
Go with it then :+1:

Reply:
In this case I will split them like that: 
``` 
<Title>Minho Covid</Title>
<Subtitle>Ajudamos a ajudar</Subtitle>
<Text>Budaerros</Text>
```
,and create the other components